### PR TITLE
Update vc-unet checkpoint to fix unmapped names; Fix SD bugs.

### PR DESCRIPTION
**Pull request by SamitHuang**
Fix unmapped names to reduce checkpoint loading time.
SD bugs in lr retrieval fixed.

### CODE REVIEW

File: examples/stable_diffusion_v2/scripts/run_train_v2_v_pred.sh
Comment (DongyaoZhu) [inline examples/stable_diffusion_v2/scripts/run_train_v2_v_pred.sh:1]: #122 will delete this file

File: examples/stable_diffusion_v2/ldm/modules/train/callback.py
Comment (DongyaoZhu) [inline examples/stable_diffusion_v2/ldm/modules/train/callback.py:1]: is there a reason?

**Review by DongyaoZhu — COMMENTED** [review]
wr

File: examples/stable_diffusion_v2/ldm/modules/train/callback.py
Comment (SamitHuang) [reply examples/stable_diffusion_v2/ldm/modules/train/callback.py:1]: It is a bug introduced from 910b adaptation https://github.com/mindspore-lab/mindone/pull/133  
leading to error "Lr iterator has not asnumpy attribte"

File: examples/stable_diffusion_v2/scripts/run_train_v2_v_pred.sh
Comment (SamitHuang) [reply examples/stable_diffusion_v2/scripts/run_train_v2_v_pred.sh:1]: I see. This is to fix bug reported by the testing team

Comment (DongyaoZhu) [reply examples/stable_diffusion_v2/scripts/run_train_v2_v_pred.sh:1]: ok I'd do: 1.include it here 2. make a comment under 122 so they include the change too

**Review by DongyaoZhu — APPROVED** [review]

**Review by wtomin — APPROVED** [review]

File: examples/videocomposer/configs/exp03_sketch2video_style.yaml [hunk 1]
```diff
@@ -16,9 +16,8 @@ batch_sizes: {
 vit_image_size: 224
 network_name: UNetSD_temporal
 resume: true
-resume_step: 228000
```
[the remark this answers is not in the document]
Comment (SamitHuang) [reply examples/videocomposer/configs/exp03_sketch2video_style.yaml:19]: Should clear all those unused args later

**Pull request merged**